packages/configs/default/index.json

```diff
@@ -31,6 +32,8 @@
     "*.html": "@parcel/packager-html",
     "*.css": "@parcel/packager-css",
     "*.js": "@parcel/packager-js",
+    "*.webmanifest": "@parcel/packager-string",
+    "manifest.json": "@parcel/packager-string",
```

Comment:
the same as in https://github.com/parcel-bundler/parcel/pull/3404#discussion_r314878203 (to discuss)